pkg/push/api/api.go

```diff
@@ -236,20 +241,149 @@ func (s *PushService) CreatePush(
 		}
 		return nil, dt.Err()
 	}
-	return &pushproto.CreatePushResponse{}, nil
+	return &pushproto.CreatePushResponse{
+		Push: push.Push,
+	}, nil
 }
 
-func (s *PushService) validateCreatePushRequest(req *pushproto.CreatePushRequest, localizer locale.Localizer) error {
-	if req.Command == nil {
-		dt, err := statusNoCommand.WithDetails(&errdetails.LocalizedMessage{
+// CreatePushV2 implement logic without command
+func (s *PushService) CreatePushV2(
```

Comment:
Is this a new API?
If so, instead of creating a new API, can you implement this using the current `CreatePush` API?

Reply:
Oh no, this is still the old API, I should have named it as createPushV2 or createPushNoCommand, this function is called in https://github.com/bucketeer-io/bucketeer/pull/1289/files#diff-927c02bd692b9ee775cb3720ede88aa16612d46cf46213f45274a0ddd792c782R114

Reply:
I have rename the function to avoid misunderstanding 😄